apps/daffio/src/app/docs/docs.module.ts

```diff
@@ -5,7 +5,7 @@ import { RouterModule } from '@angular/router';
 import { DaffContainerModule } from '@daffodil/design/container';
 import { DaffHeroModule } from '@daffodil/design/hero';
 import {
-  provideDaffDocsDesignExamples,
+  provideDaffDocsExampleComponents,
```

Comment:
@damienwebdev @griest024 

`provideDaffDocsDesignExamples` doesn't exist in the codebase, so I assumed it's supposed to be `provideDaffDocsDesignExampleComponents`. I don't know if this is correct though.